maps/src/main/res/layout/offline_map_layers_picker.xml

```diff
@@ -90,26 +89,36 @@
         android:id="@+id/layers"
         android:layout_width="0dp"
         android:layout_height="0dp"
-        android:visibility="gone"
         app:layoutManager="androidx.recyclerview.widget.LinearLayoutManager"
         app:layout_constraintStart_toStartOf="parent"
         app:layout_constraintEnd_toEndOf="parent"
-        app:layout_constraintBottom_toTopOf="@id/bottom_divider"
+        app:layout_constraintBottom_toTopOf="@id/add_layer"
         app:layout_constraintHeight_default="wrap"
-        app:layout_constraintTop_toBottomOf="@id/top_divider"
-        tools:visibility="visible" />
+        app:layout_constraintTop_toBottomOf="@id/top_divider" />
 
     <com.google.android.material.progressindicator.CircularProgressIndicator
         android:id="@+id/progress_indicator"
         android:layout_width="wrap_content"
         android:layout_height="wrap_content"
         android:layout_marginVertical="@dimen/margin_small"
         android:indeterminate="true"
-        app:layout_constraintBottom_toTopOf="@id/bottom_divider"
+        app:layout_constraintBottom_toTopOf="@id/add_layer"
         app:layout_constraintEnd_toEndOf="@id/message"
         app:layout_constraintStart_toStartOf="@id/message"
         app:layout_constraintTop_toBottomOf="@id/top_divider" />
 
+    <org.odk.collect.androidshared.ui.multiclicksafe.MultiClickSafeMaterialButton
```

Comment:
I think this needs more of a margin between the list and its top.

Reply:
Done.